### PR TITLE
readme link fix and update

### PR DESCRIPTION
**Pull request by chrisjmckeown**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,17 +8,35 @@
...
-    * On the left hand side show a history of your last 5 searches and views.
-    * In the central panel introduce and explain the website.
-    * On the right hand side show a google map with your location.
-* Search for a band by key word.
-* View the list of results on the left and listen to 30 second song previews.
-* When an artist is clicked on the central panel will display artist information
-    * The map will show event location.
+* When loaded the page will display:
+    * Search and history on the left hand panel.
+    * Informatation about the website including future development opertunities in the central panel.
```
Comment (smoneo) [inline README.md]:
```suggestion
    * Information about the website including future development oportunities in the central panel.
```

File: README.md [hunk 2]
```diff
@@ -8,17 +8,35 @@
...
+    * A Google Map set to your location on the right hand panel.
+* To search:
+    * Enter a artist/band name and click search or hit the enter key.
+    * A list of results will display below the search inputst.
+    * Click on an artist to view information about them:
+        * The central panel will display an: 
+            * Image
+            * Genres
+            * Ablums
```
Comment (smoneo) [inline README.md:29]:
```suggestion
            * Albums
```

File: README.md (hunk 1 above)
Comment (tonymelek) [reply README.md]: Information about the website including future development opportunities in the central panel.

Comment (tonymelek) [reply README.md]: Opportunities has a double 'p'

**Review by tonymelek — APPROVED** [review]
Looks Good !
Approved

**Pull request merged**